src/gazebo_pose_sniffer_plugin.cpp

```diff
@@ -108,7 +108,7 @@ void PoseSnifferPlugin::OnUpdate(common::UpdateInfo const &updateInfo) {
   }
   if (!_poses.empty()) {
     sendto(_fd, (const char *)&_poses.front(),
-           sizeof(_poses[0]) * _poses.size(), MSG_CONFIRM,
```

Comment:
Is this a change required for the macOS? or something orthogonal?

Reply:
I would say both. 
This flag was preventing the compilation as not recognized by MacOs, but later I understood that it was absolutely not necessary for my usage and had nothing to do there .